scene/gui/dialogs.cpp

```diff
@@ -324,7 +337,7 @@ void WindowDialog::_bind_methods() {
 
 WindowDialog::WindowDialog() {
 
-	//title="Hello!";
+	title = "";
```

Comment:
`String` is default initialized to `""` already, so this is not necessary.

Reply:
Lots of strings are initialized like that throughout the codebase however, a cleanup may be needed.